website/package.json

```diff
@@ -33,10 +33,10 @@
     "clear": "docusaurus clear"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.15",
-    "@docusaurus/preset-classic": "^2.0.0-beta.15",
-    "@docusaurus/theme-search-algolia": "^2.0.0-beta.15",
-    "@eightshift/frontend-libs": "^5.0.3",
+    "@docusaurus/core": "^2.0.0-beta.18",
```

Comment:
Did you check if anything broke? They break things like crazy between these beta and alpha versions 😬